packages/core/src/organization-contact/commands/handlers/organization-contact-create.handler.ts

```
@@ -73,7 +73,13 @@ export class OrganizationContactCreateHandler implements ICommandHandler<Organiz
 				throw new BadRequestException('Failed to create contact details', error.message);
 			}
 
+			if (isNotEmpty(input.projects)) {
+				input.projects.forEach((project) => {
+					delete project.members;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)